types/src/on_chain_config/mod.rs

```diff
@@ -122,7 +122,7 @@ pub trait OnChainConfig: Send + Sync + DeserializeOwned {
 }
 
 pub fn new_epoch_event_key() -> EventKey {
-    EventKey::new_from_address(&association_address(), 4)
+    EventKey::new_from_address(&association_address(), 16)
```

Comment:
Is there any particular reason for this change? I'm more curious than anything else :)

Reply:
Ahh great question. The association now creates 12 new event handles before creating the `new_epoch` handle: 4 each (burn event, preburn event, cancel burn event, mint event) for the three currencies.